loopchain/peermanager/peer_loader.py

```diff
@@ -49,13 +49,13 @@ def _load_peers_from_file(peer_manager: 'PeerManager'):
     @staticmethod
     def _load_peers_from_rest_call(peer_manager: 'PeerManager'):
         rs_client = ObjectManager().channel_service.rs_client
-        is_block_version_0_3 = PeerLoader._is_block_version_0_3(rs_client)
         crep_root_hash = conf.CHANNEL_OPTION[ChannelProperty().name].get('crep_root_hash')
 
-        if is_block_version_0_3 and crep_root_hash:
-            return PeerLoader._get_reps_by_root_hash_call(peer_manager, rs_client, crep_root_hash)
+        if not crep_root_hash:
+            logging.error(f"There's no crep_root_hash to initialize.")
```

Comment:
The channel option `crep_root_hash` is not optional but required to run loopchain node, Right?

Reply:
Yep. `crep_root_hash` is required for citizen node.